pilot/pkg/kube/inject/inject.go

```diff
@@ -715,7 +717,8 @@ func intoObject(sidecarTemplate string, meshconfig *meshconfig.MeshConfig, in ru
 		podSpec,
 		metadata,
 		meshconfig.DefaultConfig,
-		meshconfig)
+		meshconfig,
+		DefaultSidecarProxyUID)
 	if err != nil {
```

Comment:
Is it safe to use the default value here?

Reply:
This preserves the old behavior. The thing is, we can't put the UID into the configmap, because the uid that should be used differs from one namespace to the next.